### ISSUE TITLE
nc15 version no longer shows previews of folders

### ISSUE THREAD

**Issue by schniepp** [nextcloud/gallery#493]
# Feature request

**User type**: Logged-in or public user

**User level**: Beginners, intermediate or advanced

## Description
Until NC14, folder thumbnails show previews of images in the folder. Now this is completely gone and only the folder icon is shown in grid mode. This functionality is dearly missed. Please bring it back. 

## Benefit / value
It will be more efficient to browse large collections of data. Now, one needs to go into every folder to get an idea what's inside.

## Risk / caveats
None known. 

## Sponsorship

**Are you a developer willing to implement this feature?**: no

**Can you sponsor the development of this feature or do you know someone who can?**: no

**Comment by Julien-Blanc-tgcm** [comment]
An important thing to take into consideration is sharing. I use the gallery view to share some photos with friends : it is much nicer than the file view or the new mosaic view. Currently, sharing from the gallery gives a link that opens the gallery view. That should not be broken by a new selection scheme.

Also note that since the nc15 update, if someone clicks on the "folder view" button in a shared gallery, he can’t go back to the gallery view.

**Comment by setnes** [comment]
I would have logged this as a bug.  The grid view is not a good replacement for gallery view.  Eliminating the ability to switch to gallery mode from within a folder with images forces a user to navigate the entire NextCloud file tree to get back to the same folder in gallery mode.  Then... if they switch to list/detail view, they cannot get back to gallery view.  This is a big mess.... especially for private link sharing.

Grid view is for all files.  Gallery view is for images.

**Comment by github-tomster** [comment]
great example again.
where are these decisions made?
look at the 'feature' there: https://github.com/nextcloud/android/issues/3348
@schniepp 
nice report!
@who-ever-decided-that!
taken away a bit more of user-control, well done!
@world 
99% of the users wont even notice the difference...

**Comment by mzamot** [comment]
This is true, but unfortunately the gallery doesn't seems to be actively maintained.
